R/dataset.R

```diff
@@ -20,6 +20,14 @@ getNrow <- function (dataset) {
     return(nrows)
 }
 
+setFragment <- function (x, name, variable) {
```

Comment:
I don't think this function should exist. "fragment" isn't a meaningful, standard enough concept in our API such that a generic "setFragment" would be useful, at least not at this time. Just do whatever requesting in the `pk<-` method.